README.md

```diff
@@ -205,17 +205,22 @@ To add a new function to PostgreSQL by VSHN:
 entrypoint to start working with gRPC server is to run:
 ```
 go run main.go --log-level 1 start grpc --network tcp --socket ':9547' --devmode
+
 ```
 
 This will start the GRPC server listening on a TCP port. Afterward you can configure the composition to use this connection:
 
 ```yaml
+### Macos:
```

Comment:
Maybe we can re-use the readme snipped in line 99 here again?
```
HOSTIP=$(docker inspect kindev-control-plane | jq '.[0].NetworkSettings.Networks.kind.Gateway')
# HOSTIP=host.docker.internal # On some docker distributions
# HOSTIP=host.lima.internal # On lima distributions
```

That seems to cover most cases.

Reply:
ok, I added this snippet